src/main/java/by/exadel/internship/entity/User.java

```diff
@@ -38,10 +40,13 @@
     @Enumerated(EnumType.STRING)
     private UserRole userRole;
 
-    //    @Column(name = "technology")
-//    private List<Technology> techTechnology;
+    @Column(name="us_name", nullable=false)
+    @Enumerated(EnumType.STRING)
+    @ElementCollection(fetch = FetchType.EAGER)
```

Comment:
Do you really need EAGER here? 
I suggest use LAZY by default as much as possible. 

Here I expect a lot of scenarios where this list of skills will be redundant. For example to assign user for the interview you'll need to get all users with required skill. Getting all skills for each user will be redundant.

Reply:
i got it. Done.